dev/cognitiveservices/data-plane/Language/readme.md

```diff
@@ -0,0 +1,66 @@
+# Cognitive Services Language SDK
+
+This is the AutoRest configuration file the Cognitive Services Language SDK.
+
+> see https://aka.ms/autorest
+
+## Releases
+
+The current preview release of Language is 2022-07-01-preview.
+
+<!-- The current stable release of Language is 2022-05-01. -->
+
+```yaml
+tag: release_preview
+add-credentials: true
+clear-output-folder: true
+openapi-type: data-plane
+directive:
+  - suppress: LongRunningResponseStatusCode
+    reason: The validation tools do not properly recognize 202 as a supported response code.
+```
+
+### Preview Release
+
+``` yaml
+input-file:
+  - preview/analyzetext.json
+  - preview/analyzetext-authoring.json
+  - preview/analyzeconversations.json
+  - preview/analyzeconversations-authoring.json
+title:
+  Microsoft Cognitive Language Service
+modelerfour:
+  lenient-model-deduplication: true
+```
+
+<!-- ### Stable Release
+
+``` yaml
+input-file:
+  - stable/analyzetext.json
+  - ...
+title:
+  Microsoft Cognitive Language Service
+modelerfour:
+  lenient-model-deduplication: true
+``` -->
+
```

Comment:
Just get rid of this block. There's only one release based purely on files. Use branches to work in parallel on preview or stable releases as needed: what you were doing anyway - but now you can marshal changes to/from branches using familiar `git merge` or `git cherry-pick` commands.